mscore/preferences.cpp

```diff
@@ -123,6 +123,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_APP_TELEMETRY_ALLOWED,                           new BoolPreference(false, false)},
             {PREF_APP_STARTUP_TELEMETRY_ACCESS_REQUESTED,          new StringPreference("", false)},
             {PREF_APP_BACKUP_GENERATE_BACKUP,                      new BoolPreference(true)},
+            {PREF_APP_BACKUP_SUBFOLDER,                            new StringPreference("")},
```

Comment:
I disagree with the default being empty, `.backup`, `.mscbackup`, whatever, starting with a dot (to make it hidden on Linux and Mac), but not back to the clutter we had before.